integration-tests/lib/test-functions.sh

```diff
@@ -155,10 +157,9 @@ cleanup_resources() {
     echo "Cleanup log file: ${cleanup_log_file}"
     echo -e "\n--- Cleanup Log ---" > "${cleanup_log_file}"
 
-    echo "Deleting Github branch ${component_branch} and PR branch konflux-${component_branch} for repo ${component_repo_name}..." >> "${cleanup_log_file}"
-    # Ensure SUITE_DIR is available
-    "${SUITE_DIR}/../scripts/delete-single-branch.sh" "${component_repo_name}" "${component_branch}" >> "${cleanup_log_file}" 2>&1
-    "${SUITE_DIR}/../scripts/delete-single-branch.sh" "${component_repo_name}" "konflux-${component_branch}" >> "${cleanup_log_file}" 2>&1
```

Comment:
Will delete-single-branch still be used at all after this change?

Reply:
should not be used. I was going to keep around for future use. but ok with deleting too.

Reply:
IMO, because this framework is so new and not everyone has a great handle on it yet, I think go ahead with deleting unused stuff